mindjunkies/forums/views.py

```
@@ -54,7 +54,7 @@
             return self.handle_reply_submission(request, course_slug)
         # This is a react submission
         elif "reaction" in request.POST:
-            print("this is saima")
+            
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)